change/@uifabric-styling-2020-03-11-09-35-22-marygans-messageBar-color-update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "MessageBar: removed transparency in colors, updated color values, and added semantic slots to reflect designs.",
```

Comment:
Since this change file is for styling, you should probably only include the parts about semantic slot and color value updates